stable/kubernetes-dashboard/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image: gcr.io/google_containers/kubernetes-dashboard-amd64
+imageTag: "v1.6.0"
+imagePullPolicy: "IfNotPresent"
+
+namespace: kube-system
+
```

Comment:
Let's do how @seanknox suggests. Currently, `helm list` gets confused and shows the chart living in the "default" namespace although everything is deployed in kube-system.